pcsx2/gui/AppMain.cpp

```diff
@@ -563,6 +586,27 @@ void Pcsx2App::LogicalVsync()
 		}
 	}
 
+	if (g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
+		if (EnableFMV == 1) {
```

Comment:
Just ``if (EnableFMV)`` is enough. The ``== 1`` might make people think it's an ``int`` whereas it's only a ``bool``. I suggest to use the bool types ``true`` or ``false`` whenever possible when dealing with Boolean variables